generators/premiums/azure_speech_to_text/config.py

```diff
@@ -0,0 +1,15 @@
+from util.configs import build_generator_premium_config
+from util.enums import State
+from . import azure_speech_to_text, INPUT_EXAMPLE
+
+
+def get_config():
+    return build_generator_premium_config(
+        function=azure_speech_to_text,
+        input_example=INPUT_EXAMPLE,
+        data_type="text",
+        issue_id=250,
+        tabler_icon="BrandWindows",
```

Comment:
`BrandWindows` is already occupied by Microsoft translator, might wanna change that - very minor issue